x-pack/plugins/stack_alerts/public/alert_types/components/data_view_select_popover.tsx

```diff
@@ -5,48 +5,76 @@
  * 2.0.
  */
 
-import React, { useCallback, useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
 import { i18n } from '@kbn/i18n';
 import {
+  EuiButtonEmpty,
   EuiButtonIcon,
   EuiExpression,
   EuiFlexGroup,
   EuiFlexItem,
   EuiFormRow,
   EuiPopover,
+  EuiPopoverFooter,
   EuiPopoverTitle,
 } from '@elastic/eui';
 import { DataViewsList } from '@kbn/unified-search-plugin/public';
 import { DataViewListItem } from '@kbn/data-views-plugin/public';
 import { useTriggersAndActionsUiDeps } from '../es_query/util';
 
-interface DataViewSelectPopoverProps {
+export interface DataViewSelectPopoverProps {
   onSelectDataView: (newDataViewId: string) => void;
-  initialDataViewTitle: string;
-  initialDataViewId?: string;
+  dataViewName?: string;
+  dataViewId?: string;
 }
 
 export const DataViewSelectPopover: React.FunctionComponent<DataViewSelectPopoverProps> = ({
   onSelectDataView,
-  initialDataViewTitle,
-  initialDataViewId,
+  dataViewName,
+  dataViewId,
 }) => {
-  const { data } = useTriggersAndActionsUiDeps();
+  const { data, dataViewEditor } = useTriggersAndActionsUiDeps();
   const [dataViewItems, setDataViewsItems] = useState<DataViewListItem[]>();
   const [dataViewPopoverOpen, setDataViewPopoverOpen] = useState(false);
 
-  const [selectedDataViewId, setSelectedDataViewId] = useState(initialDataViewId);
-  const [selectedTitle, setSelectedTitle] = useState<string>(initialDataViewTitle);
+  const closeDataViewEditor = useRef<() => void | undefined>();
+
+  const loadDataViews = useCallback(async () => {
+    const fetchedDataViewItems = await data.dataViews.getIdsWithTitle();
+    setDataViewsItems(fetchedDataViewItems);
+  }, [setDataViewsItems, data.dataViews]);
+
+  const closeDataViewPopover = useCallback(() => setDataViewPopoverOpen(false), []);
+
+  const createDataView = useMemo(
+    () =>
+      dataViewEditor?.userPermissions.editDataView()
```

Comment:
When I create a user who can create stack rules but has no access to create data views and I have no data views available, I see this:
<img width="605" alt="Screen Shot 2022-06-30 at 1 12 03 PM" src="https://user-images.githubusercontent.com/13104637/176737704-54d94a61-9d56-4f7e-8c24-f0a090f1dba7.png">

Wondering if there should be any helper text about needing the right privileges to create data views?

Reply:
Added the following message to the popover. Wdyt @ymao1 @gchaps ?
cc @andreadelrio 

<img width="519" alt="Screenshot 2022-07-04 at 15 25 45" src="https://user-images.githubusercontent.com/1415710/177164498-aa9c842e-2fec-47d5-9a8a-e9fda21574e8.png">

Reply:
Here is my suggestion:

You need additional privileges to create data views.  Contact your administrator.

Reply:
@gchaps Thanks Gail! Updated the copy.